PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -1137,6 +1137,29 @@ SavingStatus FileManager::saveBuffer(BufferID id, const TCHAR * filename, bool i
 		}
 	}
 
+	const wchar_t* filePath = buffer->getFullPathName();

```

Comment:
Is there a chance `buffer->getFullPathName()` to be empty (in case the file is completely new and has never been written to disk before)?
Shouldn't we use `fullpath` instead the new `const wchar_t* filePath` ?

Reply:
You are right about it.